tests/autorag/nodes/passagecompressor/test_refine.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+from autorag import generator_models
+from autorag.nodes.passagecompressor.refine import refine
+from tests.autorag.nodes.passagecompressor.test_tree_summarize import queries, retrieved_contents, check_result, df
```

Comment:
Make new 'test_base_passage_compressor' python file inside test folder and save queries, df etc. at there.
Importing from other modules test file can occur error like circular import.